src/shared/helpers/sortMovies.ts

```diff
@@ -0,0 +1,9 @@
+import { Movie } from "../../models";
+
+export function sortMovies(movies: Movie[], isDownDirection: boolean) {
+  return movies.sort((movie1, movie2) =>
+    isDownDirection
+      ? movie1.release - movie2.release
+      : movie2.release - movie1.release
+  );
+}
```

Comment:
let it be this way, but on the next HW we need to be able to sort by release or name alphabet, but not down or up. There wasn't information about that.